examples/js_of_ocaml/.jshintignore

```diff
@@ -0,0 +1 @@
+js/todomvc.ml
```

Comment:
hmm wow i have never seen jshintignore but it actually seems perfect for what we want!